_entities.md

```diff
@@ -320,6 +320,24 @@ type | The type of endpoint. Possible values are 'email', 'card' and 'external'.
       "numberFormat": {
```

Comment:
Lets remove `hasOtpEnabled` from the documentation since we will not support it in the future.